src/components/products-list.vue

```diff
@@ -1,26 +1,32 @@
 
 <template>
-  <div :class="'product' + (index + 1)" v-for="(product, index) in products">
-    <img
-      :src="product.img"
-      imgfield="img"
-      style="max-width: 150px; width: 150px"
-      alt=""
-    />
+  <div :class="'product'" v-for="(product, index) in products">
+    <img :src="product.img" imgfield="img" style="max-width: 150px; width: 150px" alt="" />
     <p>{{ product.name }}</p>
+    <div class="product-item">
+      <img src="https://cdn.icon-icons.com/icons2/2785/PNG/512/shopping_cart_add_icon_177407.png" alt="Добавить в корзину"
```

Comment:
add point to show that product is already in cart
add point to remove product from cart